skglm/estimators.py

```diff
@@ -1159,6 +1162,169 @@ def fit(self, X, y):
     # TODO add predict_proba for LinearSVC
 
 
+class CoxEstimator(LinearModel):
+    r"""Elastic Cox estimator with Efron and Breslow estimate.
+
+    Refer to :ref:`Mathematics behind Cox datafit <maths_cox_datafit>`
+    for details about the datafit expression. The data convention for the estimator is
+
+    - ``X`` the design matrix with ``n_features`` predictors
+    - ``y`` a two-column vector where the first ``tm`` is of event time occurrences
+      and the second ``s`` is of censoring.
+
+    For L2 Cox, ``l1_ratio=0.``, :ref:`LBFGS <skglm.solvers.LBFGS>`
+    is the used solver, otherwise it is :ref:`ProxNewton <skglm.solvers.ProxNewton>`.
+
+    Parameters
+    ----------
+    alpha : float, optional
+        Penalty strength. It must positive ``alpha > 0``.
+
+    l1_ratio : float, default=0.7
```

Comment:
Any hint on this default?

Reply:
It should be `0.5` to align with the existing codebase and `scikit-learn`.